src/components/data/kanban/kanban.tsx

```diff
@@ -1,127 +1,72 @@
 import React, { useEffect, useState } from "react";
 
-import {
-  AttributeData,
-  DEFAULT_URL_FIELDS,
-  FieldSet,
-  field2Title,
-  formatMessage,
-  useIntl,
-} from "../../../lib";
-import {
-  GroupedAttributeDataProps,
-  getContextData,
-} from "../../../lib/data/groupedattributedata";
-import { Button, ButtonLink, ButtonLinkProps, ButtonProps } from "../../button";
+import { Button, ButtonProps } from "../../button";
 import { Select } from "../../form";
 import { Column, Grid } from "../../layout";
-import { Body, H1, H2, H3, P } from "../../typography";
+import { Body, H2, H3 } from "../../typography";
 import "./kanban.scss";
 
-export type KanbanProps = GroupedAttributeDataProps & {
-  /** If set, items are `draggable` allowing the user to rearrange them (across) columns). */
+export type KanbanProps = {
+  /** If set, items are `draggable` allowing the user to rearrange them (across columns). */
   draggable?: boolean;
-  /** The kanban "change column" (accessible) label */
+  /** The kanban-template "change column" (accessible) label */
   labelSelectColumn?: string;
 
-  /** The kanban "move object position" (accessible) label. */
+  /** The kanban-template "move object position" (accessible) label. */
   labelMoveObject?: string;
 
-  /** Get called when the fieldsets change. */
-  onFieldsetsChange?: (fieldsets: FieldSet[]) => void;
+  /** Get called when the componentList changes. */
+  onComponentListChange?: (componentList: KanbanColumn[]) => void;
 
-  /** Get called when the objectLists change. */
-  onObjectListsChange?: (objectLists: AttributeData[][]) => void;
+  /** The new componentList prop defining columns and their items */
+  componentList: KanbanColumn[];
 
-  /** Get called when an object is dropped onto a column. */
-  onObjectChange?: (
-    object: AttributeData,
-    sourceColumnIndex: number,
-    targetColumnIndex: number,
-    sourceObjectIndex: number,
-    targetObjectIndex: number,
-  ) => void;
+  /** The title of the kanban */
+  title?: string;
+};
+
+export type KanbanColumn = {
+  title: string;
+  items: React.ReactNode[];
```

Comment:
Does this mean that it is the responsibility of the component using the Kanban component to render the cards as jsx? wouldn't it make more sense to pass to the `Kanban` the information that should be displayed and then let this `Kanban` component turn this info into jsx?

Reply:
We discussed this and decided to leave it as it is for now and discuss this with @svenvandescheur next week

Reply:
This has now become the KanbanColumnItem type to be able to have an ID. We can discuss next week if there should be more properties, but it should be easy to update now.